pnpm-workspace.yaml

```diff
@@ -1,79 +1,64 @@
 packages:
-  - "packages/*"
-  - "apps/*"
+  - packages/*
+  - apps/*
+
+catalogMode: strict
```

Comment:
A [new setting](https://pnpm.io/catalogs#catalogmode) to enforce using the catalog